src/sonic-py-common/sonic_py_common/syslogger.py

```diff
@@ -4,13 +4,19 @@
 import socket
 import sys
 
+# customize python logging to support notice logger
+logging.NOTICE = logging.INFO + 1
```

Comment:
@Junchao-Mellanox how did you arrive at the formula?

Reply:
Python logging package support customizing log level. I read the source code of it and find that:
```python
#---------------------------------------------------------------------------
#
# Default levels and level names, these can be replaced with any positive set
# of values having corresponding names. There is a pseudo-level, NOTSET, which
# is only really there as a lower limit for user-defined levels. Handlers and
# loggers are initialized with NOTSET so that they will log all messages, even
# at user-defined levels.
#

CRITICAL = 50
FATAL = CRITICAL
ERROR = 40
WARNING = 30
WARN = WARNING
INFO = 20
DEBUG = 10
NOTSET = 0
```

As you can see, there are 10 spare numbers between INFO and WARN, so I take INFO+1 as the NOTICE level.